lib/pharos/phase.rb

```diff
@@ -61,5 +61,15 @@ def exec_script(script, vars = {})
     def parse_resource_file(path, vars = {})
       Pharos::YamlFile.new(resource_path(path)).read(vars)
     end
+
+    # @return [Hash]
+    def mem_storage
```

Comment:
Not sure what is the proper name for this?

Reply:
I'll offer `ClusterContext`

Reply:
`def cluster_context` ? Sounds good to me.

Reply:
Yeah, `cluster` because it's used to store things that are not node-specific... might also consider some kind of `host_context` later, for things like the `cpu_arch` which are currently set on the `Pharos::Configuration::HostNode` by `ValidateHost`.

Reply:
About arch: yes, it's a bit problematic.. currently we don't really support clusters with different archs.